pytest_repo_health/plugin.py

```diff
@@ -118,6 +119,20 @@ def pytest_ignore_collect(path, config):
             return True
     return False
 
+def pytest_collection_modifyitems(session, config, items):
+    # TODO(jinder): add: and session.config.getoption("repo_health_metadata")
+    if session.config.getoption("repo_health"):
+        checks_metadata = {}
+        for item in items:
+            if '__pytest_repo_health__' in dir(item._obj):
+                if item.parent.name not in checks_metadata.keys():
+                    checks_metadata[item.parent.name] = {}
+                    checks_metadata[item.parent.name]['module_doc_string'] = item.parent._obj.__doc__.strip()
+                checks_metadata[item.parent.name][item.name] = item._obj.__pytest_repo_health__
+                checks_metadata[item.parent.name][item.name]["doc_string"] = item._obj.__doc__.strip()
```

Comment:
Nit: Could clean up this section to have less repetition, but maybe not worth the time at the moment?

Reply:
I decrease a little bit, but more refractoring not worth it for now